docs/pages/info/trains.mdx

```diff
@@ -4,7 +4,7 @@ name: Trains
 
 Unlike in many other APLs, trains in TinyAPL are written using special syntax: <hl code="⦅" />, <hl code="⋄" /> and <hl code="⦆" />. They are a sequence of arrays, functions, adverbs and conjunctions that combine to create functions or modifiers.
 
-Note that, if a train has no diamonds, the contents are parsed with a special syntax where function application doesn't exist (but modifier application does), and then each tree becomes a tine. This means that in most cases, you can omit diamonds in trains:&nbsp;<hl code="⦅+-×⦆" />&nbsp;for&nbsp;<hl code="⦅+⋄-⋄×⦆" />. When you want to use an empty tine, you can use the special token `·` instead.
+Note that, if a train has no diamonds, the contents are parsed with a special syntax where function application doesn't exist (but modifier application does), and then each tree becomes a tine. This means that in most cases, you can omit diamonds in trains:&nbsp;<hl code="⦅+-×⦆" />&nbsp;for&nbsp;<hl code="⦅+⋄-⋄×⦆" />.
```

Comment:
Can you keep a mention to the name "compact trains"? I use it a lot and i don't think "trains without diamonds" flows as nice